flake.nix

```diff
@@ -0,0 +1,53 @@
+{
+  description = "RealWorld spec in the PureScript Halogen framework";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-21.11";
+    easy-purescript-nix = {
+      url = "github:justinwoo/easy-purescript-nix";
+      flake = false;
+    };
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
+  };
+
+  outputs = { self, nixpkgs, easy-purescript-nix, ... }@inputs:
+    let
+      name = "halogen-realworld";
+
+      supportedSystems = [
+        "aarch64-darwin"
+        "x86_64-darwin"
+        "x86_64-linux"
+      ];
+
+      forAllSystems = f: nixpkgs.lib.genAttrs supportedSystems (system: f system);
+    in
+    {
+      devShell = forAllSystems (system:
+        let
+          pkgs = import nixpkgs { inherit system; };
+
+          easy-ps = import easy-purescript-nix { inherit pkgs; };
+        in
+        pkgs.mkShell {
+          inherit name;
+          buildInputs = (with pkgs; [
+            nodejs-16_x
+            nixpkgs-fmt
```

Comment:
What do you think of `alejandra`?

Reply:
See section 2 of the original comment: https://github.com/thomashoneyman/purescript-halogen-realworld/pull/113#pullrequest-930670291